tests/unit_tests/ensemble_evaluator/test_scheduler.py

```diff
@@ -0,0 +1,71 @@
+import asyncio
+import json
+import logging
+from pathlib import Path
+
+import pytest
+
+from ert.ensemble_evaluator import Monitor, identifiers, state
+from ert.ensemble_evaluator.config import EvaluatorServerConfig
+from ert.ensemble_evaluator.evaluator import EnsembleEvaluator
+
+
+@pytest.mark.timeout(60)
+def test_scheduler_receives_checksum_and_waits_for_disk_sync(
+    tmpdir, make_ensemble_builder, monkeypatch, caplog, run_monitor_in_loop
+):
+    num_reals = 1
+    custom_port_range = range(1024, 65535)
+    done_renaming = asyncio.Event()
+
+    async def rename_and_wait():
+        Path("real_0/job_test_file").rename("real_0/test")
+        done_renaming.set()
+        await asyncio.sleep(5)
```

Comment:
I guess you could give `rename_and_wait`  an event to wait for instead of the sleep which you then set once `"Waiting for disk syncronization"` is logged ?